src/main/proxy-functions/shell-api-request.ts

```diff
@@ -60,9 +63,17 @@ export class ShellRequestAuthenticator extends Singleton {
     }
 
     const authToken = clusterTokens.get(tabId);
+    const buf = Uint8Array.from(token.split(",").map(ar => {
+      const res = +ar;
+
+      if (isNaN(res)) {
+        throw new TypeError("Token must be a CSV list of digits");
```

Comment:
could the token come in as base64 instead? Could be cleaner to handle?

Reply:
Probably, let me see

Reply:
Done